src/system.h

```diff
@@ -906,6 +906,12 @@ typedef struct {
     const Char *    cookie;
 } StructGVarFunc;
 
+#define GAP_GVAR_FUNC_TABLE_ENTRY(srcfile, name, nparam, params) \
+  {#name, nparam, \
+   params, \
+   (GVarFunc)name, \
+   srcfile ":Func" #name }
```

Comment:
As I argued yesterday one could make the `Func` prefix an enforced policy in the kernel.

Of course this means that one needs to have this bit of information to be able to browse the kernel code.